tests/dragonfly/sentinel_test.py

```diff
@@ -5,6 +5,18 @@
 import pytest
 import time
 import asyncio
+from datetime import datetime
+from sys import stderr
+
+# logs messages to stderr, including a timestamp.
+# use log() instead of print().
+def log(*args, **kwargs):
```

Comment:
I thought we'll be using something like https://docs.python.org/3/howto/logging.html - and generally have something that is more reusable to all other tests.
To that end if we indeed us the logging lib I think it'd be nice to set it up with a config file so it is easily reusable.
What do you think?

Reply:
https://docs.pytest.org/en/7.1.x/how-to/logging.html

Reply:
Thanks for the comments!
Originally I did consider `logging` but deemed it to not be very helpful, only offering printing to `stdout, stderr` using a cumbersome interface.

I'll take another look, and write a quick configuration to use with Pytest.